app-text/cherrytree/cherrytree-0.99.52.ebuild

```diff
@@ -1,23 +1,25 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
-PYTHON_COMPAT=( python3_{7..10} )
+PYTHON_COMPAT=( python3_{8..11} )
 
 inherit cmake python-any-r1 xdg
 
 DESCRIPTION="A hierarchical note taking application (C++ version)"
-HOMEPAGE="https://www.giuspen.com/cherrytree https://github.com/giuspen/cherrytree"
+HOMEPAGE="https://www.giuspen.com/cherrytree/"
 
 if [[ ${PV} == *9999 ]]; then
 	inherit git-r3
 	EGIT_REPO_URI="https://github.com/giuspen/cherrytree"
 	S="${WORKDIR}/${P}/future"
 else
-	SRC_URI="https://github.com/giuspen/cherrytree/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+	SRC_URI="https://github.com/giuspen/cherrytree/releases/download/${PV}/${P/-/_}.tar.xz"
+	S="${WORKDIR}"/${P/-/_}
+
 	KEYWORDS="~amd64 ~x86"
-# 	S="${WORKDIR}/${PN}-${COMMIT}/future"
+	# S="${WORKDIR}/${PN}-${COMMIT}/future"
 fi
```

Comment:
That package does not have any live-ebuild so you can proceed in 2 ways: drop that if...else...fi or create (and maintain) the live-ebuild

Reply:
I'm not sure there's anything wrong with keeping dead live-specific code in the ebuild as it provides an easy point from which to make a live ebuild if desired in the future. No reason to require that being done now.